pages/index.js

```diff
@@ -6,10 +6,11 @@ import Link from 'next/link';
 export default function Home() {
   return (
     <>
-      <Title>Welcome to RPG 4-you</Title>
+      <Title>RPG Sheet-Creator</Title>
```

Comment:
try not to change code that is not related to the pull request. In this case you wanted to add a navigation but edited something on the main page, which is not related.